libmscore/lyrics.cpp

```diff
@@ -384,6 +384,8 @@ void Lyrics::layout2(int nAbove)
 
       if (placeBelow()) {
             qreal yo = segment()->measure()->system()->staff(staffIdx())->bbox().height();
+            //TODO: get real height from staffType(tick)- for staff changes (but involves autoplace decisions)
+            //  if (staffType()) { yo = staffType()->lines() * staffType()->lineDistance().val); }
```

Comment:
There are some places on the code that compute positions and autoplacing from the unchanged staff height boundaries. This could be adapted to the real staffType(tick) boundaries but that would be a separate PR.